targets/ar71xx-generic/profiles.mk

```diff
@@ -134,3 +134,12 @@ $(eval $(call GluonModel,WZRHPG450H,wzr-hp-g450h-squashfs,buffalo-wzr-hp-g450h))
 # WZR-HP-AG300H/WZR-600DHP
 $(eval $(call GluonProfile,WZRHPAG300H))
 $(eval $(call GluonModel,WZRHPAG300H,wzr-hp-ag300h-squashfs,buffalo-wzr-hp-ag300h-wzr-600dhp))
+
+## Netgear
+
+# WNDR3700v2/3800
+$(eval $(call GluonProfile,WNDR3700))
+$(eval $(call GluonProfileFactorySuffix,WNDR3700,.img))
+$(eval $(call GluonModel,WNDR3700,wndr3700-squashfs,netgear-wndr3700))
+$(eval $(call GluonModel,WNDR3700,wndr3700v2-squashfs,netgear-wndr3700v2))
```

Comment:
Please add a comment that the 3700v2 will advertise itself as a 3800, and that the auto updater will thus use the 3800 image (and that this is okay as the 3700v2 and 3800 sysupgrade images are identical)